src/ItemNote.ts

```diff
@@ -182,6 +182,11 @@ const tagsToNoteContent = (
 // excerpt do not mess up front matter
 const normalizeTitle = (excerpt: String) =>
   excerpt.replace(/---./g, "").replace(/"/g, "'");
+
+const normalizedate = (dateinput:number) =>{
+  const date = new Date(dateinput*1000)
```

Comment:
1000 is a magic number. Can you add a comment that explains it or assign it to variable that explains what it means? I prefer the latter.

Reply:
The new Date(milliseconds-input) creation will transform the time to readable date which is milliseconds-input of milliseconds after January 1, 1970, 00:00:00 UTC.

However, the pocket API would return the date of my recent item with a 10-figure number, which would transform to some date in the year of 1970. In addition, there is no formal explanation in the API documents [Pocket Developer Program: Pocket API: Retrieve](https://getpocket.com/developer/docs/v3/retrieve).

After some trial, I just found out that recent dates would transform to 13-figure numbers, I guess that pocket API returns time in **seconds** rather than milliseconds. Hence I just put 1000 there and it worked.

Since this is somewhat constant, setting it to a variable might not be useful. So, I think I could put a short comment above the to the function.

Reply:
On the other hand, it would be nice if date-format can be specified by users. But I'm quite new to obsidian and just learned javascript in last 3 days, it's a bit hard for me now XD.

I'll try to figure out hot to use setting panel. If I can crack that, I think more specifications could be added, like: 
- get ALL items rather than unread items (by adding state:"all" to request options, default is "unread", could also be "archive")
- get item of last month (by adding since:"timestamp" to request options)

Reply:
Hm I think it's fine as it is now, because it's simple. It already provides value, and the additional functionality can be built in a future PR if needed.

Reply:
Sure, I'll dig into that later and made a new PR after a test it.